SU-HSDO/suhumsci
# ARCH-72 event card template and styles, new mixins, latest decanter and bourbon

# READY FOR REVIEW

# Summary
- Horizontal Card, Vertical Card, and Date Stacked Vertical Card Twig, SCSS (and CSS), and UI Components files
- New card (with custom display for horizontal layout) and accent dash mixins
- Updated package.json to include latest versions of Decanter and Bourbon
- Custom Archaeology styles and updates to the Gruntfile.js as needed

# Needed By (Date)
- End of sprint (6/21)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the /patterns page on local shows a horizontal, vertical, and date stacked card that can be used for patterns on the mockups at https://invis.io/MHLC2CFGQYV#/299456614_Homepage and https://projects.invisionapp.com/d/main#/console/14029316/292457736/preview

# Affected Projects or Products
- Archaeology
- suhumsci

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-72
- https://stanfordits.atlassian.net/browse/ARCH-83
- https://stanfordits.atlassian.net/browse/ARCH-81

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
